db-drda/src/main/java/com/splicemachine/db/drda/NetworkServerControl.java

```diff
@@ -273,9 +273,38 @@ public NetworkServerControl(String userName, String password)
 	 */
 	public NetworkServerControl(InetAddress address,int portNumber) throws Exception
 	{
-		serverImpl = new NetworkServerControlImpl(address, portNumber);
+		this(address, portNumber, "");
 	}
 
+	/**
+	 *
+	 * Creates a NetworkServerControl object that is configured to control
+	 * a Network Server on a  specified port and InetAddress.
+	 *<P>
+	 * <B> Examples: </B>
+	 * </P>
+	 * <P>
+	 * To configure for port 1621 and listen on the loopback address:
+	 *<PRE>
+	 *  NetworkServerControl  util = new
+	 * NetworkServerControl(InetAddress.getByName("localhost"), 1621);
+	 * </PRE>
+	 * </P>
+	 *
+	 * @param address     The IP address of the Network Server host.
+	 *                     address cannot be null.
+
+	 * @param portNumber  port number server is to used. If <= 0,
+	 *                    default port number is used
+	 *
+	 * @throws             Exception on error
+	 */
+	public NetworkServerControl(InetAddress address,int portNumber, String externalHostname) throws Exception
```

Comment:
Why `InetAddress` not simply a `String`? Why a separate `portNumber` if it could be part of the InetAddress? Could this `InetAddress` use a port that is different from the `portNumber`?  Could we change `portNumber` to `port`?